### DISCUSSION TITLE
Canonical Tag - Url

### DISCUSSION THREAD

**Discussion by Stefa093** [vercel/next.js#49673] [Help]
### Summary

Hey guys how can I mark that my url is canonical on the new metadata?

**Comment by darrylmorley** [comment]
If you have set the 'metadataBase' in your layout, then:

```
export const metadata = {
return {
   alternates: {
        canonical: '/',
      },
}}
```

This is according to the docs and Stack Overflow:

https://stackoverflow.com/questions/75938020/how-to-set-canonical-tag-in-nextjs-13
https://nextjs.org/docs/app/api-reference/functions/generate-metadata

**Reply by mrxsal** [reply]
setting `canonical: '/'` always sets the canonical to the root url for me: `http://127.0.0.1:3000/`

The following works: `canonical: './'`
